src/callbacks/onLayout/initStudyDayControl.js

```diff
@@ -0,0 +1,82 @@
+import { extent } from 'd3';
+import startTransition from './initStudyDayControl/startTransition';
+import stopTransition from './initStudyDayControl/stopTransition';
+
+export default function initStudyDayControl() {
+    var chart = this;
+    var config = this.config;
+    const studyDayControlWrap = this.controls.wrap
+        .selectAll('div')
+        .filter(controlInput => controlInput.label === 'Study Day');
+
+    chart.controls.studyDayInput = studyDayControlWrap.select('input');
+
+    //convert control to a slider
+    chart.controls.studyDayInput.attr('type', 'range');
+
+    //set min and max values and add annotations
+    chart.controls.studyDayRange = extent(chart.imputed_data, d => d[config.studyday_col]);
+    chart.controls.studyDayInput.attr('min', chart.controls.studyDayRange[0]);
+    chart.controls.studyDayInput.attr('max', chart.controls.studyDayRange[1]);
+    chart.controls.studyDayInput.attr('step', 7);
+
+    studyDayControlWrap
+        .insert('span', 'input')
+        .attr('class', 'span-description')
+        .style('display', 'inline-block')
+        .style('padding-right', '0.2em')
+        .text(chart.controls.studyDayRange[0]);
+    studyDayControlWrap
+        .append('span')
+        .attr('class', 'span-description')
+        .style('display', 'inline-block')
+        .style('padding-left', '0.2em')
+        .text(chart.controls.studyDayRange[1]);
+
+    //initialize plot_day to day 0 or the min value, whichever is greater
+    if (config.plot_day === null) {
+        config.plot_day = chart.controls.studyDayRange[0] > 0 ? chart.controls.studyDayRange[0] : 0;
+        chart.controls.studyDayInput.node().value = config.plot_day;
```

Comment:
broken record: `value` here gets round to the nearest 7 counting from the minimum study day